cmd/querydsltest/querydsltest.go

```diff
@@ -0,0 +1,72 @@
+package main
```

Comment:
This `querydsltest` thing is basically just my test binary, but I'm keeping it in the repo for now, in part because it shows off what's possible.